### PR TITLE
Added API stubs

### LINKED ISSUE #74: Create API Stubs
**Describe the task that needs to be done.**
API stubs need to be set up to return mock data to different parts of the app.

**Describe how a solution to your proposed task might look like (and any alternatives considered).**
Will set up API stubs for communication, pairing, timetable and user, based off the backend. This will allow parts of the app to call these functions and use mock data.

**Notes**
N/A

### PR DESCRIPTION
**Pull request by sv-singh**
## Description
This PR adds API stubs for communication, pairing, timetable and user, based off the backend. These new functions can now be called in different parts of the app and will return mock data.

This fixes #74 

## How has this been tested?
Called the functions in parts of the app to see that the correct mock data is getting returned.

## Screenshots
N/A

## Checklist
*(Leave blank if not applicable)*

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/api/CommunicationAPI.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+ * Sends a message from one user to another
+ * @param {string} sender id of the sender
+ * @param {string} receiver id of the receiver
+ * @param {string} content content of the message
+ * @return {Promise} data entry of the message sent
+ */
+export const sendMessage = async ({ sender, receiver, content }) => {
+  const timestamp = Date.now();
+  console.log(sender, receiver, content, timestamp);
+};
```
Comment (NicholasBatesNZ) [inline src/api/CommunicationAPI.js:40]: While we (Comms team) are able to implement this endpoint if desired, we were thinking a SocketIO event would suffice. See https://github.com/SE701-T1/backend/pull/72

Comment (scorpionknifes) [reply src/api/CommunicationAPI.js:40]: Think this was added cause it exists in the backend communication controller. I'm happy to remove this, we just need to deprecate or remove the endpoint in the backend.

Comment (NicholasBatesNZ) [reply src/api/CommunicationAPI.js:40]: Right that makes sense. My PR does remove it from the controller :+1:

Comment (sv-singh) [reply src/api/CommunicationAPI.js]: Sure I've removed it

**Review by scorpionknifes — APPROVED** [review]

**Pull request merged**